credential/cargo-credential/Cargo.toml

```diff
@@ -12,3 +12,7 @@ serde = { workspace = true, features = ["derive"] }
 serde_json.workspace = true
 thiserror.workspace = true
 time.workspace = true
+
+[dev-dependencies]
+snapbox = { workspace = true, features = ["examples"] }
+tempfile.workspace = true
```

Comment:
Not seeing this used (and snapbox has some tempfile support)

Reply:
Thanks